setup.py

```diff
@@ -121,5 +122,4 @@ def get_version(*file_paths):
         "License :: OSI Approved :: Apache Software License",
     ],
     install_requires=load_requirements('requirements/base.in'),
-    tests_require=load_requirements('requirements/test.in'),
```

Comment:
Removed this because of this warning `/openedx/src/django-pyfs/.tox/quality/lib/python3.11/site-packages/setuptools/_distutils/dist.py:270: UserWarning: Unknown distribution option: 'tests_require'
  warnings.warn(msg)`